projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "event_queue.h"
+#include "gpio.h"
+#include "status.h"
+
+// lights_gpio requires lights_gpio_config to be initialized first.
```

Comment:
Please update your comments. I don't think this is necessary anymore.

Also, we generally follow the scheme:

```
#pragma once
// Module Summary
// Requires ...
//
// Module description - intent, intended usage, etc.
#include ...
```